app/javascript/application.js

```diff
@@ -1,6 +1,7 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
+import "loading_spinner"
 
 // Show the progress bar after 200 milliseconds, not the default 500
-window.Turbo.setProgressBarDelay(200);
+Turbo.config.drive.progressBarDelay = 200;
```

Comment:
Ha, I think I opened a separate ticket for this yesterday so you can grab it, move it to the Sprint, and Done it when this merges.

https://mitlibraries.atlassian.net/browse/USE-166

Reply:
Lol I didn't bother mentioning it on the other PR since I knew I had it here. :)